docs/api-docs/storefront/graphql/graphql-carts-and-checkout.mdx

```diff
@@ -530,6 +531,48 @@ If using the getCart query with customer impersonation tokens, you must include
   </Tab>
 </Tabs>
 
+### Create a cart redirect URL
+
+<Callout type="info">
+You can use a cart redirect URL for embedded checkouts or to redirect your checkout. You need a `cartEntity` ID. To create a `cartEntity` ID, use the [create a cart](#Create-a-cart-using-a-simple-product) mutation. 
+</Callout>
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+  ```graphql filename="Example mutation: Create cart redirect URL" showLineNumbers copy
+  mutation{
+   cart {
+       createCartRedirectUrls(input:{
+           cartEntityId: "dc5e7b24-0047-4b3e-b16b-2cdc90943b86"}) {
+           redirectUrls{
+               embeddedCheckoutUrl
+               redirectedCheckoutUrl
+           }
+       }
+  
+   }
+}
+```
+  </Tab>
+  <Tab>
+    
+```json filename="Example mutation: Create cart redirect URL" showLineNumbers copy
```

Comment:
This code block needs to be moved one (to the right). You had it right on [#L542](https://github.com/bigcommerce/docs/pull/289/files#diff-6ff391d937e95e1d2821796b1c639c1ca0fb9f698e63a570ff895188a2c6d7b5R542)